alternator/executor.cc

```diff
@@ -1120,6 +1145,10 @@ static future<executor::request_return_type> create_table_on_shard0(tracing::tra
     if (tags && tags->IsArray()) {
         update_tags_map(*tags, tags_map, update_tags_action::add_tags);
     }
+    if (bm.provisioned) {
+        tags_map[RCU_TAG_KEY] = std::to_string(bm.rcu);
+        tags_map[WCU_TAG_KEY] = std::to_string(bm.wcu);
+    }
```

Comment:
This patch only modifies CreateTable, what about UpdateTable?
And in CreateTable it only sets these tags for the base table - what about the GSIs/LSIs?

All these different cases should have tests written for them.